src/scss/reviews.scss

```diff
@@ -0,0 +1,127 @@
+$white-background: #ffffff;
+$main-background: #081130;
+
+.reviews__section {
+  background: $main-background;
+
+  padding: 80px 0;
+}
+
+.reviews__title {
+  margin-bottom: 50px;
+
+  color: #ffffff;
+  text-align: center;
+}
+
+.user-card {
+  display: flex;
+
+  padding: 70px 140px;
+
+  background-color: $white-background;
+  box-shadow: 2px 8px 16px rgba(194, 198, 208, 0.06);
+  border: 2px solid #E2EAF2;
+  border-radius: 5px;
+}
+
+.user-card-review {
+  width: 60%;
+  margin-right: 54px;
+}
+
+.user-card-info {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  width: 40%;
+}
+
+.user-card-review__title {
+  display: flex;
+  flex-direction: column;
+
+  margin-bottom: 24px;
+
+  color: #2E363E;
+  font-weight: 500;
+  font-size: 1.875rem;
+  line-height: 143%;
+
+  &:before {
+    content: "❝";
+
+    margin: 0 0 24px 0;
+
+    font-size: 5rem;
+    color: #5D6AFB;
+  }
+}
+
+.user-card-review__text {
+  display: inline-block;
+
+  margin: 30px 0 24px 0;
+
+  color: #2E363E;
+  font-size: 0.9375rem;
+  line-height: 170%;
+
+}
+
+.user-card-review__link {
+  display: inline-block;
+
+  font-size: 0.875rem;
+  line-height: 21px;
+  color: #4A94FF;
+
+  border-bottom: 1.5px dashed #4A94FF;
+}
+
+.user-card-info__image {
+  width: 380px;
+  height: 380px;
+  border-radius: 50%
+}
+
+.user-card-info__title {
+  color: #2E363E;
+  font-size: 1.3125rem;
+  font-weight: 500;
+  line-height: 143%;
+  margin-top: 30px;
+}
+
+.user-card-info__subtitle {
+  font-size: 1rem;
+  line-height: 178%;
+  color: #A6AEB6;
+  font-weight: 400;
+}
+
+.reviews-arrows__container {
+  margin-top: 70px;
+  text-align: center;
+  .reviews__arrow-btn {
+    width: 2em;
+    height: 2em;
+
+    background-color: #FED84C;
```

Comment:
Should be a variable.

Reply:
![image](https://user-images.githubusercontent.com/78299696/212350373-95ba9a43-4f68-44ab-9dd9-af49469a0c1e.png)